docs/pre-release/doc-links.yml

```diff
@@ -56,6 +54,10 @@
        link: reference/dns
      - title: RBAC
        link: reference/rbac
+     - title: Networking through Virtual Network Interface
+       link: reference/tun-device
+     - title: Connection Routing
+       link: reference/routing
```

Comment:
I guess we accidentally removed this from the docs / never linked it, good catch!